README.md

```diff
@@ -210,106 +185,91 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
 <!-- markdownlint-disable-next-line -->
 
-> [!NOTE]
-> ⚠️ = implemented but no test vectors present
+### Decentralized Identifiers
 
-### [`did:web`](https://w3c-ccg.github.io/did-method-web/)
+#### [DID Documents](https://www.w3.org/TR/did-core/) & [DID Resolution](https://w3c-ccg.github.io/did-resolution/)
 
-| Feature      | Typescript | Kotlin | Rust | Swift |
-| ------------ | ---------- | ------ | ---- | ----- |
-| `Resolution` | ❌          | ❌      | ⚠️    | ❌     |
+Independent of DID Methods, we support DID Documents according to [DID Core v1.0](https://www.w3.org/TR/2022/REC-did-core-20220719/) with the adjustments specified in [this corresponding document](spec/did.md).
 
-### [`did:jwk`](https://github.com/quartzjer/did-jwk/blob/main/spec.md)
+#### DID Methods
 
-| Feature      | Typescript | Kotlin | Rust | Swift |
-| ------------ | ---------- | ------ | ---- | ----- |
-| `Creation`   | ❌          | ❌      | ⚠️    | ❌     |
-| `Resolution` | ❌          | ❌      | ⚠️    | ❌     |
+| Method                                                 | Creation   | Resolution | Note |
+| ------------------------------------------------------ | ---------- | ---------- | ---- |
+| [`did:web`](https://w3c-ccg.github.io/did-method-web/) | ❌         | ✅         | -   |
+| [`did:jwk`](https://github.com/quartzjer/did-jwk/blob/main/spec.md) | ✅         | ✅  | - |
+| [`did:dht`](https://did-dht.com)                       | ✅         | ✅         | This is our default method. |
+| [`did:key`](https://w3c-ccg.github.io/did-method-key/) | ⚠️         | ⚠️         | Has been implemented in both Kotlin and Typescript, no plans for support in other languages. |
+| [`did:ion`](https://identity.foundation/sidetree/spec) | ⚠️         | ⚠️         | Support for `did:ion` has been deprecated. |
 
-### [`did:dht`](https://tbd54566975.github.io/did-dht-method/)
+### Verifiable Credentials
 
-| Feature      | Typescript | Kotlin | Rust | Swift |
-| ------------ | ---------- | ------ | ---- | ----- |
-| `Creation`   | ⚠️          | ⚠️      | ❌    | ❌   |
-| `Resolution` | ⚠️          | ⚠️      | ❌    | ❌   |
+#### [W3C Verifiable Credential Data Model 1.1](https://www.w3.org/TR/vc-data-model)
 
-### [`did:key`](https://w3c-ccg.github.io/did-method-key/)
+We support Verifiable Credentials according to the [Verifiable Credentials Data Model 1.1](https://www.w3.org/TR/vc-data-model) with the adjustments specified in [this corresponding document](spec/vc.md).
 
-| Feature      | Typescript | Kotlin | Rust | Swift |
-| ------------ | ---------- | ------ | ---- | ----- |
-| `Creation`   | ⚠️          | ⚠️      | ⚠️    | ❌   |
-| `Resolution` | ⚠️          | ⚠️      | ⚠️    | ❌   |
+| Feature                                                          |
+| ---------------------------------------------------------------- |
+| Data model                                                       |
+| Validation (data model, JSON Schema, status)                     |
```

Comment:
Does `status` here refer to this https://www.w3.org/TR/vc-data-model-2.0/#status? (wondering why it's called out specifically)

Reply:
oh I see because status is a property which is a set of specs (the status lists below) as opposed to just another property of the data model

Reply:
yes...